src/views/HomeView.vue

```diff
@@ -35,7 +35,7 @@
                       :status="game.status"
                       :num-players="game.numPlayers"
                       :is-ranked="game.isRanked"
-                      @error="handleError"
+                      @error="handleSubscribeError(game.id, $event)"
```

Comment:
You’re sending the event but then calling it message in the method? That seems kind of confusing to me.

Reply:
So the event payload is the error message. It seemed more descriptive for the method argument to describe what the event is rather than just calling it `$event` inside of `handldeSubscribeError()`. Do you disagree? Do you think we should call it `$event` in both places?

We're stuck using $event in the event handler here in the html, so if we want them to be the same I would think we'd have to call both of them `$event`. But isn't it more clear to have the method argument say what the event _is_?

Reply:
Is it the name `message` that's specifically confusing? Would it be better as `errMessage`?